target/bin/fail2ban

```diff
@@ -3,6 +3,23 @@
 # shellcheck source=../scripts/helper-functions.sh
 . /usr/local/bin/helper-functions.sh
 
+if ! iptables -L &>/tmp/iptables-check-command-output
+then
+  echo "IPTables does not seem to work properly. The output of 'iptables -L' was:
+
+$(</tmp/iptables-check-command-output)
```

Comment:
```suggestion
if ! IPTABLES_OUTPUT=$(iptables -L 2>&1)
then
  echo "IPTables does not seem to work properly. The output of 'iptables -L' was:

${IPTABLES_OUTPUT}
```

Reply:
I didn't even know this was possible 😂👌🏽 Nice!